h5pxblock/h5pxblock.py

```diff
@@ -297,8 +351,8 @@ def result_handler(self, data, suffix=''):
 
         if self.has_score and data['result'] and data['result']['score']:
             grade_dict = {
-                'value': data['result']['score']['raw'],
-                'max_value': data['result']['score']['max'],
+                'value': data['result']['score']['raw']/data['result']['score']['max'] * self.points,
```

Comment:
nit: @BryanttV I think we should handle division by zero exception here.

Reply:
@ziafazal updated